src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -221,24 +217,26 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
   }
 
   private _handlePadClick(e: MouseEvent): void {
+    const input = this.shadowRoot!.querySelector(
+      "#alarmCode"
+    ) as PaperInputElement;
```

Comment:
You can use the decorator `@query` to create a getter with the querySelector and have it available easier: `@query("#alarmCode") private _input: PaperInputElement;`
You can then use: `this._input`

Reply:
That sounds like a great idea! 👍  Will implement it asap